lib/node_modules/@stdlib/ndarray/base/some-by/README.md

```diff
@@ -0,0 +1,216 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2025 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# someBy
+
+> Test whether at least `n` elements in an ndarray pass a test implemented by a predicate function.
+
+<section class="intro">
+
+</section>
+
+<!-- /.intro -->
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var someBy = require( '@stdlib/ndarray/base/some-by' );
+```
+
+#### someBy( arrays, predicate\[, thisArg ] )
+
+Tests whether at least `n` elements in an ndarray pass a test implemented by a predicate function.
+
+<!-- eslint-disable max-len -->
+
+```javascript
+var Float64Array = require( '@stdlib/array/float64' );
+var scalar2ndarray = require( '@stdlib/ndarray/base/from-scalar' );
+
+function predicate( value ) {
+    return value > 0.0;
+}
+
+// Create a data buffer:
+var xbuf = new Float64Array( [ 1.0, 2.0, 3.0, 4.0, 5.0, 0.0, 7.0, 8.0, 9.0, 10.0, 11.0, 12.0 ] );
+
+// Define the shape of the input array:
+var shape = [ 3, 1, 2 ];
+
+// Define the array strides:
+var sx = [ 4, 4, 1 ];
+
+// Define the index offset:
+var ox = 0;
+
+// Create the input ndarray-like object:
+var x = {
+    'dtype': 'float64',
+    'data': xbuf,
+    'shape': shape,
+    'strides': sx,
+    'offset': ox,
+    'order': 'row-major'
+};
+
+var n = scalar2ndarray( 3.0, 'generic', 'row-major' );
+
+var out = someBy( [ x, n ], predicate );
+// returns true
+```
+
+The function accepts the following arguments:
+
+-   **arrays**: array-like object containing an input ndarray and a zero-dimensional ndarray.
```

Comment:
a zero-dimensional ndarray containing what? You need to be more descriptive here. Telling me that I should provide a zero-dimensional ndarray without telling me what that value is is not particularly insightful.